bundle/libraries/same_name_libraries_test.go

```diff
@@ -0,0 +1,92 @@
+package libraries
+
+import (
+	"context"
+	"testing"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/bundle/config"
+	"github.com/databricks/cli/bundle/config/resources"
+	"github.com/databricks/cli/libs/diag"
+	"github.com/databricks/databricks-sdk-go/service/compute"
+	"github.com/databricks/databricks-sdk-go/service/jobs"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSameNameLibraries(t *testing.T) {
+	b := &bundle.Bundle{
+		Config: config.Root{
+			Resources: config.Resources{
+				Jobs: map[string]*resources.Job{
+					"test": {
+						JobSettings: &jobs.JobSettings{
+							Tasks: []jobs.Task{
+								{
+									Libraries: []compute.Library{
+										{
+											Whl: "full/path/test.whl",
+										},
+									},
+								},
+								{
+									Libraries: []compute.Library{
+										{
+											Whl: "other/path/test.whl",
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+
+	diags := bundle.Apply(context.Background(), b, CheckForSameNameLibraries())
+	require.Len(t, diags, 1)
+	require.Equal(t, diag.Error, diags[0].Severity)
+	require.Equal(t, "Duplicate local library name", diags[0].Summary)
```

Comment:
Could you use local-only acceptance test here?

Reply:
Unfortunately not as it requires running bundle deploy in script which is not supported yet

Reply:
ah, got it. 

BTW @kanterov is working on having "bundle deploy" support in acceptance test https://github.com/databricks/cli/pull/2254

Reply:
This is the correct PR -- https://github.com/databricks/cli/pull/2299, it is ready to be merged. Perhaps you can check if this allows you adding acceptance tests with "bundle deploy" instead?

Reply:
Indeed, it would enable acceptance test here as well. But I'd rather follow up with the acceptance test then wait for it to be merged (unless it happens soon). Not sure if you consider this blocking though

Reply:
https://github.com/databricks/cli/pull/2299 was merged, it should be possible to test bundle deploy in acceptance tests now.

Reply:
Added acc test